core-concepts/state.md

```diff
@@ -186,7 +180,7 @@ store.$patch({
 })
 ```
 
-However, some mutations are really hard or costly to apply with this syntax: any collection modification (e.g. pushing, removing, splicing an element from an array) requires you to create a new collection. Because of this, the `$patch` method also accepts a function to group this kind of mutations that are difficult to apply with a patch object:
+Sin embargo, algunas mutaciones son muy difíciles o costosas de aplicar con esta sintaxis: cualquier modificación a una colección (por ejemplo: añadir, borrar, empalmar un elemento desde un array) requiere que crees una nueva colección. Debido a esto, el método `$patch` también acepta una función para agrupar este tipo de mutaciones que son difíciles de aplicar con un objeto de parche:
```

Comment:
In LATAM, It is not widely used "empalmar" , but we can replace it with "unir un elemento desde un array". Do you agree?

Reply:
sure :D